### ISSUE TITLE
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Admin issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#806]
**Describe the bug**
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Admin issue.

**To Reproduce**

1. Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test2&mode=readonly&name=index.html&path=%2FSites%2Ftest&pathType=page)
2. Click a site.
3. Roles, User and Categories tabs are accessible through Tab and Arrow key but not indicated.

**Expected behavior**
if Functionality of the screen is operable through Tab and arrow kays then indication should be there.

**Additional context**
Navigate all the options available for similar issue and fix.
